Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1559,6 +1563,23 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             nullptr, // A null string forces the default value.
             readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:enable-webrtc:
+     *
+     * Enable or disable support for WebRTC on pages. Enabling this setting implies MediaStream will
+     * be enabled as well.
```

Comment:
I would split the second phrase after the period to a new paragraph, so `gi-docgen` will pick only “Enable WebRTC support for loaded pages” as the property summary, otherwise the summary will be too long. We can also link to the `enable-media-stream` property:

```suggestion
     * Enable WebRTC support for loaded pages.
     *
     * Enabling this setting implies that [property@Settings:enable-media-stream]
     * will be enabled as well.
```

Reply:
I didn't know we're supposed to use gi-docgen syntax now. Isn't that compatible with gtkdoc syntax?

Reply:
Yes, since [249161@main](https://commits.webkit.org/249161@main). There's a good amount of compatibility with gtk-doc syntax indeed, but it would be good to already use documentation style more suitable for gi-docgen in new code.